docs/sources/setup-grafana/configure-security/configure-authentication/generic-oauth/index.md

```diff
@@ -14,305 +14,194 @@ weight: 700
+| `empty_scopes`               | No       | Set to `true` to use an empty scope during authentication.                                                                                                                                     | `false`         |
+| `allow_sign_up`              | No       | Whether to allow new Grafana user creation through generic OAuth2 login. If set to `false`, then only existing Grafana users can log in with generic OAuth.                                    | `true`          |
+| `auto_login`                 | No       | Set to `true` to allow users to automatically log in and skip the login screen. This setting is ignored if multiple auth providers are configured to use auto login.                           | `false`         |
+| `id_token_attribute_name`    | No       | Name of the key used to extract the ID token from the returned OAuth2 token.                                                                                                                   | `id_token`      |
+| `login_attribute_path`       | No       | [JMESPath](http://jmespath.org/examples.html) expression to use for user login lookup from the user ID token.                                                                                  |                 |
+| `name_attribute_path`        | No       | [JMESPath](http://jmespath.org/examples.html) expression to use for user name lookup from the user ID token. This name will be used as user's display name.                                    |                 |
+| `email_attribute_name`       | No       | Name of the key to use for user email lookup within the generic OAuth2 attribute map.                                                                                                          | `email:primary` |
+| `role_attribute_path`        | No       | [JMESPath](http://jmespath.org/examples.html) expression to use for Grafana role lookup.                                                                                                       |                 |
+| `role_attribute_strict`      | No       | Set to `true` to deny user login if Grafana role cannot be extracted using `role_attribute_path`.                                                                                              | `false`         |
+| `allow_assign_grafana_admin` | No       | Set to `true` to enable automatic sync of Grafana server administrator role.                                                                                                                   | `false`         |
+| `skip_org_role_sync`         | No       | Set to `true` to stop automatically syncing user roles. This will allow you to manually set organization roles for your users from within Grafana.                                             | `false`         |
```

Comment:
```suggestion
| `skip_org_role_sync`         | No       | Set to `true` to stop automatically syncing user roles. This will allow you to set organization roles for your users from within Grafana manually.                                             | `false`         |
```